package.json

```diff
@@ -26,6 +27,8 @@
     "url": "https://github.com/versatica/JsSIP/issues"
   },
   "dependencies": {
+    "@types/debug": "^4.1.5",
+    "@types/node": "^13.13.5",
```

Comment:
`@types/debug` must be in `dependencies` because JsSIP exports a `debug` instance. But why is `@types/node` there? It should be in `devDependencies`.

Reply:
Because some classes extends from EventEmitter. And if `@types/node` were in `devDependencies`, types for `EventEmitter` wouldn't be installed

Reply:
This is true, but note that JsSIP uses the [events](https://github.com/Gozala/events) package, so we should have `@types/events` in `dependencies` rather than `@types/node`, right?

Reply:
This package realize Node interface, so it's correct

Reply:
I checked it with Typescript

Reply:
It may work, but if JsSIP runs in browser we don't need Node types. Can you try by uninstalling it and installing `@types/events` instead?

Reply:
I already tried. Create empty project, install `events` and `@types\events`, create test file:
```jsx
import {EventEmitter} from 'events';

export class Test extends EventEmitter {}
```
then compiled it, and receive next declaration:
```jsx
/// <reference types="node" />
import { EventEmitter } from 'events';
export declare class Test extends EventEmitter {
}
```

Reply:
Not super expert in TS here. So what? is it ok or not?

Reply:
It's ok

Reply:
`@types/node` and `@types/events` declaration are the same

Reply:
Let's add both them because JsSIP works in browser and Node.

Reply:
It doesn't matter, because TS use only `@types/node`. And it's just interface

Reply:
If I add `@types/node`, then it will be installed, but not used

Reply:
So, what we gona do? Leave `@type/node` or replace to `@types/events`?

Reply:
@ibc are you forget about me? 🙂

Reply:
We did not, it's just very bad days. Please let's come back to this next week. We don't forget.

Reply:
ok, no problem

Reply:
if @types/node includes events definitions then lets use the first one only :+1: